test/components/viral/system/modal_component_test.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+require 'application_system_test_case'
+
+module System
+  class ModalComponentTest < ApplicationSystemTestCase
+    test 'with title' do
+      title = 'This is a modal title'
+      visit('rails/view_components/modal_component/default')
+      within('.Viral-Preview > [data-controller-connected="true"]') do
+        assert_text title
```

Comment:
**chore (blocking):** Add `assert_accessible` call within the `within` blocks on both these tests.  Accessibility is currently failing on the modals.

Reply:
Please see 680529043f3d0e344215004ae09a7632a2c3fb6c.

Reply:
Thanks for adding the `assert_accessible`